pkgs/development/python-modules/aiohttp/default.nix

```diff
@@ -29,8 +29,8 @@ buildPythonPackage rec {
 
   checkInputs = [ pytest gunicorn pytest-mock async_generator pytestrunner pytest-timeout ];
 
-  propagatedBuildInputs = [ attrs chardet multidict async-timeout yarl ]
-    ++ lib.optional (pythonOlder "3.7") idna-ssl;
+  propagatedBuildInputs = [ attrs chardet multidict async-timeout yarl idna-ssl ];
+    #++ lib.optional (pythonOlder "3.7") idna-ssl;
```

Comment:
This line correspond to aiohttp 3.4.4 setup.py [`idna-ssl>=1.0; python_version<"3.7"`](https://github.com/aio-libs/aiohttp/blob/v3.4.4/setup.py#L94).

appdaemon is failing because depend on [aiohttp 2.3.10](https://github.com/nixos/nixpkgs/blob/e1ad1a0aa2ce6f9fd951d18181ba850ca8e74133/pkgs/servers/home-assistant/appdaemon.nix#L8) which has [`idna-ssl>=1.0.0`](https://github.com/aio-libs/aiohttp/blob/58be7e2ea72f1bad64858e81ddcaaa31f0d4b4dd/setup.py#L67). I'm inclined to fix it in appdaemon expression 
```diff
diff --git i/pkgs/servers/home-assistant/appdaemon.nix w/pkgs/servers/home-assistant/appdaemon.nix
index b3df4a481cd..8f865f11db6 100644
--- i/pkgs/servers/home-assistant/appdaemon.nix
+++ w/pkgs/servers/home-assistant/appdaemon.nix
@@ -10,6 +10,7 @@ let
           inherit version;
           sha256 = "8adda6583ba438a4c70693374e10b60168663ffa6564c5c75d3c7a9055290964";
         };
+        propagatedBuildInputs = oldAttrs.propagatedBuildInputs ++ [ super.idna-ssl ];
       });
 
       yarl = super.yarl.overridePythonAttrs (oldAttrs: rec {
```

Reply:
Yes, let's do that instead.